health/health_log.c

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 
 #include "health.h"
+#include "../aclk/agent_cloud_link.h"
```

Comment:
does this belong into this PR?

Reply:
Will be removed -- it will return when we add code to handle alarms